tools/fmt_diff/dune

```diff
@@ -0,0 +1,5 @@
+(executable
+  (name        fmt_diff)
```

Comment:
I think `name` is not needed when it's the same as `public_name`

Reply:
I have to upgrade `(lang dune <N>)` from 1.0 to 1.1 for that.